db/migrations/manual/20220211-partially-adj-sythea-rel1-runout.sql

```diff
@@ -0,0 +1,10173 @@
+-- This SQL script is for bookkeeping and auditing purposes
+-- This script can be used to add the improved synthea data to opensbx
+
+BEGIN;
+DO $$
+DECLARE Dev cclf_files.id%TYPE;
+DECLARE Small cclf_files.id%TYPE;
+DECLARE Large cclf_files.id%TYPE;
+
+-- Adding three rows into cclf_files for three new CCLF files associated to synthea (improved data)
+BEGIN
+INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST990.ZC8Y21.D210929.T1834180', 'TEST990', '2021-09-29', 21, 'Completed', 0) RETURNING id INTO Dev;
+INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST991.ZC8Y21.D210929.T1834180', 'TEST991', '2021-09-29', 21, 'Completed', 0) RETURNING id INTO Small;
+INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST992.ZC8Y21.D210929.T1834180', 'TEST992', '2021-09-29', 21, 'Completed', 0) RETURNING id INTO Large;
```

Comment:
Nit: I'd recommend updating the name fields here to reflect the pattern difference between runout and regular files (see [here](https://github.com/CMSgov/bcda-app/pull/567), tl;dr is replace Y with R e.g. T.BCD.TEST992.ZC8**R**21.D210929.T1834180)

Reply:
Oh interesting, thanks @katyasoup I didn't know that about the format

Reply:
@ericbuckley np! it's easy to miss and the `type` field does the heavy lifting, but it is helpful to be able to cross-reference that field as well